### DISCUSSION TITLE
How do I get the Sudo in the windows10?

### DISCUSSION THREAD

**Discussion by SohamTilekar** [microsoft/sudo#64] [Q&A]
How do I Get the sudo command access in the windows 10?

**Comment by zadjii-msft** [comment]
For now - patience. We're working on bringing sudo back to Windows 10, but it'll take a little while to dot our i's. 

For now, follow #2 for updates ☺️

**Reply by SohamTilekar** [reply]
I want to access the current dev alpha or beta version.
